build-system/task-runner/install-amp-task-runner.js

```diff
@@ -22,18 +22,42 @@ const ampCliRunner = 'build-system/task-runner/amp-cli-runner.js';
 async function installAmpTaskRunner() {
   const npmBinDir = getStdout('npm bin --global').trim();
   const ampBinary = path.join(npmBinDir, 'amp');
-  const ampBinaryExists = await fs.pathExists(ampBinary);
-  if (ampBinaryExists) {
-    const ampBinaryIsAScript = !(await fs.lstat(ampBinary)).isSymbolicLink();
-    if (ampBinaryIsAScript) {
-      log(green('Detected'), cyan('amp'), green('task runner.'));
-      return;
-    }
-  }
+
   log(yellow('Installing'), cyan('amp'), yellow('task runner...'));
-  await fs.remove(ampBinary);
-  await fs.copy(ampCliRunner, ampBinary);
+  await fs.copy(ampCliRunner, ampBinary, {overwrite: true});
   log(green('Installed'), cyan('amp'), green('task runner.\n'));
+
+  log(
+    yellow('Auto-complete is available for the'),
+    cyan('amp'),
+    yellow('command:')
+  );
+  log(
+    '⤷ Run',
+    cyan('amp --setup_autocomplete'),
+    'to enable shell auto-complete.\n'
+  );
+  if (process.env.SHELL?.includes('bash') && process.platform === 'darwin') {
+    log(
+      yellow('macOS requires'),
+      cyan('bash-completion'),
+      yellow('to be installed for auto-complete to function on Bash:')
+    );
+    log(
+      '⤷ Run',
+      cyan('brew install bash-completion'),
+      'to install it using',
+      cyan('Homebrew') + '.'
+    );
+    log(
+      '⤷ Run',
+      cyan(
+        'echo "[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion" >> ~/.bash_profile'
```

Comment:
After running this command, launching a new terminal window merely opens `bash_completion` in a text editor instead of actually enabling completion. Let's try to figure this out once we're both in the office.